ROADMAP.md

```diff
@@ -1,8 +1,19 @@
 # Roadmap
 
+## An even better tool for development environments
+
+Compose is a great tool for development environments, but it could be even better. For example:
+
+- [Compose could watch your code and automatically kick off builds when something changes.](https://github.com/docker/fig/issues/184)
```

Comment:
+1 for @dnephin's comment, there are a lot of tools available that provide this functionality so it seems unnecessary to add it to Compose.
Maybe it's a good idea to add a suggestion how to do this to the FAQ?